libexec/alluxio-config.sh

```diff
@@ -84,7 +84,7 @@ elif [[ -n "${ALLUXIO_UNDERFS_ADDRESS}" ]]; then
 fi
 
 if [[ -n "${ALLUXIO_WORKER_MEMORY_SIZE}" ]]; then
```

Comment:
ALLUXIO_WORKER_MEMORY_SIZE -> ALLUXIO_WORKER_RAMDISK_SIZE, or maybe we should keep both env var until 3.0 @calvinjia , what's our deprecation policy?

Reply:
@yuzhu ping

Reply:
done